.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
 matrix:
   include:
     - python: '3.6'
-      env: ORANGE="3.16.0"
+      env: ORANGE="3.20.0"
```

Comment:
Wouldn't it be better to test against the latest version? `Table.concatenate` is changed in 3.24; 3.20 wouldn't require the below changes, AFAIK:

Reply:
The matrix includes also the latest release and master.